test/tests/functional/pbs_pbsnodes.py

```diff
@@ -183,11 +183,21 @@ def test_pbsnodes_av(self):
         j.set_sleep_time(1)
         jid = self.server.submit(j)
         self.server.accounting_match("E;%s;" % jid)
-        prev = self.server.status(NODE, 'last_used_time')[0]['last_used_time']
+        if self.mom.is_cpuset_mom():
+            prev = self.server.status(NODE,
+                                      'last_used_time')[1]['last_used_time']
+        else:
+            prev = self.server.status(NODE,
+                                      'last_used_time')[0]['last_used_time']
```

Comment:
Suggestion: Do this if statement, but set an index variable 'i' to 0 or 1.  Then you won't have to break the status() line across two lines.  You also won't have to do the if statement again down below since you'll already have set 'i'